improve instruction set for replacing the favicon (#1000)
strapi/documentation: docs/developer-docs/latest/development/admin-customization.md

#### Favicon

Replacing the Strapi favicon with a custom favicon.
:::tip

> This same process may be used to replace the login logo `AuthLogo` and menu logo with `MenuLogo`. For more details, please visit Strapi [logos subsection](#logos)

Reviewer: ```suggestion
This same process may be used to replace the login logo (i.e. `AuthLogo`) and menu logo (i.e. `MenuLogo`) (see [logos customization documentation](#logos)).
```

:::

1. (_optional_) Create a `./src/admin/extensions/` folder if the folder does not already exist.
2. Upload your favicon into `src/admin/extensions/`.
3. Replace the **favicon.ico** at`Strapi app root` with custom favicon.ico
4. Update your `src/admin/app.js` with the following:
```js
// path: src/admin/app.js

import favicon from './extensions/favicon.png';

export default {
  config: {
         // replace favicon with a custom icon
         head: {
                favicon: favicon,
        },
  }
}
```

> 5. Rebuild, launch and revisit your Strapi app `yarn build && yarn develop`

Reviewer: ```suggestion
5. Rebuild, launch and revisit your Strapi app `yarn build && yarn develop`.
```

> :::note

Reviewer: ```suggestion
::: tip
```

> Be certain that the cached favicon is cleared. It can be cached in your web browser and also with your domain management tool like Cloudflare's CDN

Reviewer: ```suggestion
Be certain that the cached favicon is cleared. It can be cached in your web browser and also with your domain management tool like Cloudflare's CDN.
```

#### Favicon

Replacing the Strapi favicon with a custom favicon.
:::tip

> This same process may be used to replace the login logo `AuthLogo` and menu logo with `MenuLogo`. For more details, please visit Strapi [#logos](#)

Reviewer: This link's target is empty. Which section of the documentation would you like to link to?

Author: Hi, the link works as intended. It brings you to the #configuration-options.

Please double check this; I am using Firefox latest on Ubuntu 22.04. 

https://documentation-bsk2xvgex-strapijs.vercel.app/developer-docs/latest/development/admin-customization.html#customization-options

That link brings you to the same place the [#logos](#customization-options). Only difference is, you don't need the absolute URL because you are already on that page.

Author: ohh i see, that is a typo on another line that is similar. I fixed it. Good catch.

:::

> 1. Create an extensions folder at:

Reviewer: ```suggestion
1. (_optional_) Create a `./src/admin/extensions/` folder if the folder does not already exist.
```

> `src/admin/extensions/`

Reviewer: ```suggestion
```

This line can be deleted as it's best to combine it with the previous one (same for all the examples below)

> 2. Upload your favicon into:

Reviewer: ```suggestion
2. Upload your favicon into `src/admin/extensions/`.
```

Reviewer: Please join the 2 lines, thank you :-)

`src/admin/extensions/`

> 3. Replace the **favicon.ico** at:

Reviewer: same here, please join the 2 lines
